tests/dns/dns-rrtype/test.rules

```diff
@@ -0,0 +1,8 @@
+# Only alert on requests.
+alert dns any any -> any any (dns.rrtype:1; flow:to_server; sid:1; rev:1;)
+
+# Only alert on responses.
+alert dns any any -> any any (dns.rrtype:1; flow:to_client; sid:2; rev:1;)
+
+# Negated rrtype value
+alert dns any any -> any any (dns.rrtype:!1; flow:to_client; sid:3; rev:1;)
```

Comment:
Kudos on finding a pcap in which this rule would also alert :P
Although having rules that alert for both `1` and `!1` feels a bit confusing maybe for a real-life example, I suppose it doesn't hurt as an example case :)